public/app/features/datasources/state/navModel.ts

```diff
@@ -39,7 +39,7 @@ export function buildNavModel(dataSource: DataSourceSettings, plugin: GenericDat
     }
   }
 
-  if (pluginMeta.includes && hasDashboards(pluginMeta.includes)) {
+  if (pluginMeta.includes && hasDashboards(pluginMeta.includes) && contextSrv.hasRole('Admin')) {
```

Comment:
The only access control for plugins is roles and we should only add this nav if user has the org role "Admin"